.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+# Related Issue/Addition to code
+- Issue goes here
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# Proposed Changes
+- change 1
+- change 2
+
+# Additional Info
+- any additional info for context
+
+## Checklist:
+
+- [ ] My code follows the style guidelines of this project and have read CONTRIBUTING.md
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] Any dependent changes have been merged and published in downstream modules
+- [ ] I have checked my code and corrected any misspellings
+
+# Screenshots
+
+Original | Updated
+:----------------------:|:-----------:
+** original screenshot  | ** updated screenshot **
```

Comment:
Screenshots is probably not a section that's needed for nodejs/node.

Reply:
This is for the website, not nodejs/node

Reply:
my bad, 100% thought I was in nodejs/node :)